pkgs/development/libraries/immer/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "immer";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "arximboldi";
+    repo = "immer";
+    rev = "v${version}";
+    sha256 = "sha256-R0C6hN50eyFSv10L/Q0tRdnUrRvze+eRXPrlAQsddYY=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  dontBuild = true;
+  dontUseCmakeBuildDir = true;
+
+  meta = with lib; {
+    description = "Postmodern immutable and persistent data structures for C++ — value semantics at scale";
+    homepage = "https://sinusoid.es/immer";
```

Tools (e.g., `nix-update`) can pick-up the changelog entry and add it to the commit message.

```suggestion
    homepage = "https://sinusoid.es/immer";
    changelog = "https://github.com/arximboldi/immer/releases/tag/v${version}";
```

---

Done. Thanks!